concepticondata/conceptlists/BirketSmith-1928-510.tsv

```diff
@@ -0,0 +1,511 @@
+BirketSmith-1928-510-420	420	Perhaps	2278	PERHAPS	Particles
+BirketSmith-1928-510-421	421	Thank you			Particles
+BirketSmith-1928-510-422	422	When (past)			Particles
+BirketSmith-1928-510-423	423	When (future)			Particles
+BirketSmith-1928-510-424	424	Why	1270	WHY	Particles
+BirketSmith-1928-510-425	425	Yes	1268	YES	Particles
+BirketSmith-1928-510-426	426	Yet, still			Particles
+BirketSmith-1928-510-427	427	Answers him	1275	ANSWER	Verbs
+BirketSmith-1928-510-428	428	Arrives	1387	ARRIVE	Verbs
+BirketSmith-1928-510-429	429	Asks him	1309	ASK (INQUIRE)	Verbs
+BirketSmith-1928-510-430	430	Asks st. of him	1828	ASK (REQUEST)	Verbs
+BirketSmith-1928-510-431	431	Bad, it is			Verbs
+BirketSmith-1928-510-432	432	Big, it is			Verbs
+BirketSmith-1928-510-433	433	Birth, gives	1195	GIVE BIRTH	Verbs
+BirketSmith-1928-510-434	434	Blows			Verbs
+BirketSmith-1928-510-435	435	Boils st. with a lamp			Verbs
+BirketSmith-1928-510-436	436	Boils st. with a fire			Verbs
+BirketSmith-1928-510-437	437	Builds a house			Verbs
+BirketSmith-1928-510-438	438	Builds a kayak			Verbs
+BirketSmith-1928-510-439	439	Buys st.	1869	BUY	Verbs
+BirketSmith-1928-510-440	440	Chews	321	CHEW	Verbs
+BirketSmith-1928-510-441	441	Covers a boat with skin			Verbs
+BirketSmith-1928-510-442	442	Cures him	499	CURE	Verbs
+BirketSmith-1928-510-443	443	Cuts st. in pieces			Verbs
+BirketSmith-1928-510-444	444	Dies	1494	DIE	Verbs
+BirketSmith-1928-510-445	445	Drifts (snow)			Verbs
+BirketSmith-1928-510-446	446	Drills	1501	BORE	Verbs
+BirketSmith-1928-510-447	447	Drinks	1401	DRINK	Verbs
+BirketSmith-1928-510-448	448	Drums	111	DRUMMING	Verbs
+BirketSmith-1928-510-449	449	Drums in a certain way			Verbs
+BirketSmith-1928-510-450	450	Eats	1336	EAT	Verbs
+BirketSmith-1928-510-451	451	Feels st.			Verbs
+BirketSmith-1928-510-452	452	Feels cold			Verbs
+BirketSmith-1928-510-453	453	Fishes	2638	FISHING	Verbs
+BirketSmith-1928-510-454	454	Follows him	1085	FOLLOW	Verbs
+BirketSmith-1928-510-455	455	Forgets him			Verbs
+BirketSmith-1928-510-456	456	Good, it is			Verbs
+BirketSmith-1928-510-457	457	Harpoons a sea mammal			Verbs
+BirketSmith-1928-510-458	458	Hears	1408	HEAR	Verbs
+BirketSmith-1928-510-459	459	Helps him	1897	HELP	Verbs
+BirketSmith-1928-510-460	460	Hunts caribou			Verbs
+BirketSmith-1928-510-461	461	Hunts seal in kayak			Verbs
+BirketSmith-1928-510-462	462	Hunts seal at breathing hole			Verbs
+BirketSmith-1928-510-463	463	Hunts seal crawling on the ice			Verbs
+BirketSmith-1928-510-464	464	Kills st.	1417	KILL	Verbs
+BirketSmith-1928-510-465	465	Laughs	1355	LAUGH	Verbs
+BirketSmith-1928-510-466	466	Lies (falsehood)	601	LIE (MISLEAD)	Verbs
+BirketSmith-1928-510-467	467	Lights a fire			Verbs
+BirketSmith-1928-510-468	468	Obeys	1635	OBEY	Verbs
+BirketSmith-1928-510-469	469	Paddles	3668	PADDLE (MOVEMENT)	Verbs
+BirketSmith-1928-510-470	470	Pays him	718	PAY	Verbs
+BirketSmith-1928-510-471	471	Plays	1413	PLAY	Verbs
+BirketSmith-1928-510-472	472	Presents him with st.			Verbs
```

Comment:
1039 SHOW? Though might be a bit too broad.

Reply:
Yeah, I think SHOW is a tad bit too broad.